packages/ckb/src/rgbpp/btc-time.ts

```diff
@@ -117,25 +116,17 @@ export const buildBtcTimeCellsSpentTx = async ({
   return ckbTx;
 };
 
-/**
- * Sign the BTC time cells spent transaction with Secp256k1 private key
- * @param secp256k1PrivateKey The Secp256k1 private key of the master address
- * @param ckbRawTx The CKB raw transaction to be signed
- * @param collector The collector that collects CKB live cells and transactions
- * @param masterCkbAddress The master CKB address
- * @param outputCapacityRange(Optional) [u64; 2], filter cells by output capacity range, [inclusive, exclusive]
- * @param ckbFeeRate(Optional) The CKB transaction fee rate, default value is 1100
- * @param isMainnet True is for BTC and CKB Mainnet, false is for BTC and CKB Testnet
- */
-export const signBtcTimeCellSpentTx = async ({
-  secp256k1PrivateKey,
+export const completeBtcTimeCellSpentTx = async ({
```

Comment:
Suggestion: `completeBtcTimeCellSpentTx` -> `buildBtcTimeCellSpentTx`

Reply:
I'm not sure if this change is a good idea since the suggested function name is very similar to `buildBtcTimeCellsSpentTx`, which might cause confusion.

Reply:
Yes, you are right. Maybe `prepareBtcTimeCellSpentUnsignedTx` is better?